adapters/handlers/rest/configure_api.go

```diff
@@ -999,11 +999,18 @@ func registerModules(appState *state.State) error {
 			Debug("enabled module")
 	}
 
-	if _, ok := enabledModules[modmulti2vecpalm.Name]; ok {
-		appState.Modules.Register(modmulti2vecpalm.New())
+	_, enabledMulti2VecGoogle := enabledModules[modmulti2vecgoogle.Name]
+	_, enabledMulti2VecPaLM := enabledModules[modmulti2vecgoogle.LegacyName]
+	if enabledMulti2VecGoogle || enabledMulti2VecPaLM {
+		appState.Modules.Register(modmulti2vecgoogle.New())
 		appState.Logger.
 			WithField("action", "startup").
-			WithField("module", modmulti2vecpalm.Name).
+			WithField("module", modmulti2vecgoogle.Name).
+			Debug("enabled module")
+		appState.Modules.Register(modmulti2vecgoogle.NewWithLegacyName())
```

Comment:
is it needed to register both modules or it could just be enough to register `appState.Modules.Register(modmulti2vecgoogle.New())` but probably logging that was used the legacy name

Reply:
That was also a point that was raised by Andrzej during a review of my previous PR where I renamed the text2vec-palm module. Module system has a builit in support for alternative names but unfortunately that when I have implemented an `AltNames()` interface I was surprised that this feature only half worked :( there was one thing (class settings) that wasn't working and that's why I have abandoned this approach in favor of registering this module for the second time but with another name. Once all of the renaming PR's are merged I will devote 1 day and try to fix the `AltNames()` implementation so that maybe I would get rid of this second register